### DISCUSSION TITLE
Is it possible to specify the distance between two points by X?

### DISCUSSION THREAD

**Discussion by alex4998** [Live-Charts/LiveCharts2#325] [Q&A]
Let's say I have 1000 points. It is too much to display all of them at once on the CartesianChart. The chart looks too compressed. I understand I can specify MinLimit and MaxLimit on the Axis object and display let's say from point 0 to point 100 but in this case the chart looks too loose. To find the right number of points that I can properly fit the chart I have to do certain calculations. I need to know the width of the chart in screen coordinates and I need to figure out how many points I can display on the given width to make it look good and then specify MinLimit and MaxLimit. All that boils down to finding how many points I need to display to have a particular distance between each two close points in screen coordinates.
   I wonder maybe I'm missing something and all that already exists. If it doesn't exist where would be the right place to implement such calculation and functionality to maintain the constant distance between two close point?

**Comment by SuperDaveOsbourne** [comment]
How about changing the width of your bars slightly narrower? It might be easier for the math to work out on sizing them.  Also changing the padding of the control slightly?

**Reply by alex4998** [reply]
I cannot say that I fully understood the idea. Do you have a quick example? Where to put it is another question. Is subclassing Axis and putting it there is a good idea? I'd like to keep that distance between two next points constant even when the width of the chart is changed. The default behavior is when you change the chart view width the chart inside is compressed or expanded. Also, judging by no other ideas I assume that there is no such functionality out of the box. Frankly speaking I'm quite surprised.
